src/frontend/reducers/index.js

```diff
@@ -1,5 +1,10 @@
-import { combineReducers } from 'redux'
-
-export default combineReducers({
-  ui: state => ''
-});
\ No newline at end of file
+export default function counter(state = {}, action) {
```

Comment:
what's `counter` ? This reducer should not be here, move it do dedicated file and only import in index